runtime/nls/dump/j9dmp.nls

```diff
@@ -598,4 +598,18 @@ J9NLS_DMP_JIT_TRACE_IL_CRASHED_THREAD.system_action=The JVM continues.
 J9NLS_DMP_JIT_TRACE_IL_CRASHED_THREAD.user_response=Diagnostic information, provide this information to your service representative.
 J9NLS_DMP_JIT_TRACE_IL_CRASHED_THREAD.link=
 
-# END NON-TRANSLATABLE
\ No newline at end of file
+# END NON-TRANSLATABLE
+
+J9NLS_DMP_PROCESSING_DETAILED_EVENT_TIME=Processing dump event \"%1$s\", detail \"%3$.*2$s\" \"%5$.*4$s\" at %6$s - please wait.
```

Comment:
Suggest the exception message should be introduced, perhaps as
```
J9NLS_DMP_PROCESSING_DETAILED_EVENT_TIME=Processing dump event \"%1$s\", detail \"%3$.*2$s\", exception \"%5$.*4$s\" at %6$s - please wait.
```

Please don't modify nls files for other languages, we leave that to our translation team.

Reply:
@keithc-ca The problem with modifying the message is it changes message for other event as well. For example, on heap oom:
`
JVMDUMP055I Processing dump event "systhrow", detail "java/lang/OutOfMemoryError", exception "Java heap space" at 2022/10/06 00:47:12 - please wait.`

So I kept it as similar to the current message. I think even other event could use the same message. What do you suggest?

Reply:
I wasn't suggesting that we modify `J9NLS_DMP_PROCESSING_EVENT_TIME`. I was suggesting that `J9NLS_DMP_PROCESSING_DETAILED_EVENT_TIME` (a new message) should have something better than just a space between the "detail" and the exception message.

Reply:
I don't understand the point of the last comment: As I read it, it just summarizes the purpose of this pull request.

Reply:
Apologies for the confusion. Let me try to rephrase. 

If I do this modification to the new nls message:

`J9NLS_DMP_PROCESSING_DETAILED_EVENT_TIME=Processing dump event \"%1$s\", detail \"%3$.*2$s\", exception \"%5$.*4$s\" at %6$s - please wait.`

This will also change the message for other events such as OOM, as these events will use the new message instead of existing `J9NLS_DMP_PROCESSING_EVENT_TIME` : 

_# java -Xmx1m GenerateOOM
[...]
JVMDUMP055I Processing dump event "systhrow", detail "java/lang/OutOfMemoryError", exception "Java heap space" at 2022/10/06 00:47:12 - please wait._

Requesting your suggestion if I should modify the new nls message J9NLS_DMP_PROCESSING_DETAILED_EVENT_TIME, or keep it as it is with a space between detail and exception message.

Reply:
Yes, the addition of `, exception` (where you currently have just a space) is what I'm proposing.

The only place I see where this message (or the new version) is used is in `trigger.c`. My understanding is that the intent of this change is to improve such messages (so change is inevitable). If that is problematic, then I'm not sure what else we can do beside close this PR.

Reply:
Thank you so much for your help Keith! Modified the message.